pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -248,6 +248,11 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation) *gardencorev1
 			Fn:           flow.TaskFn(botanist.WaitUntilKubeAPIServerReady).SkipIf(o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(deployKubeAPIServer),
 		})
+		deploySeedLogging = g.Add(flow.Task{
```

Comment:
Is something speaking against deploying it right after the creation of the shoot namespace and the generation of the secrets? How is it dependent on the kube-apiserver?

Reply:
Hmmm you are correct. Loki depends on the seed's Apiserver because it fetches the logs metadata from it, but does not depend on the shoot apiserver

Reply:
👍 Then let's deploy it as early as possible, i.e., after the secrets are generated and ready